pkg/ibmvpc/examples/out/analysis_out/filters_split_lb_subnet_all_vpcs__no_lbAbstract.txt

```diff
@@ -0,0 +1,98 @@
+vsi0-sub3[10.240.128.4] => alb[Fake LB private IP][10.240.1.4] : All Connections
+vsi0-sub3[10.240.128.4] => alb[Fake LB private IP][10.240.129.128] : All Connections
+vsi0-sub3[10.240.128.4] => alb[Fake LB private IP][10.240.65.128] : All Connections
+vsi0-sub3[10.240.128.4] => alb[LB private IP][10.240.65.4] : All Connections
+vsi0-sub3[10.240.128.4] => vsi0-ctrl-sub2[10.240.66.4] : All Connections
+vsi0-sub3[10.240.128.4] => vsi0-sub2[10.240.64.5] : All Connections
+vsi0-sub3[10.240.128.4] => vsi1-sub1[10.240.0.4] : All Connections
+vsi0-sub3[10.240.128.4] => vsi1-sub2[10.240.64.4] : All Connections
+vsi0-sub3[10.240.128.4] => vsi1-sub3[10.240.128.5] : All Connections
+vsi1-sub1[10.240.0.4] => alb[Fake LB private IP][10.240.1.128] : All Connections
+vsi1-sub1[10.240.0.4] => alb[Fake LB private IP][10.240.1.4] : All Connections
+vsi1-sub1[10.240.0.4] => alb[Fake LB private IP][10.240.129.128] : All Connections
+vsi1-sub1[10.240.0.4] => alb[Fake LB private IP][10.240.65.128] : All Connections
+vsi1-sub1[10.240.0.4] => alb[LB private IP][10.240.65.4] : All Connections
+vsi1-sub1[10.240.0.4] => vsi0-ctrl-sub2[10.240.66.4] : All Connections
+vsi1-sub1[10.240.0.4] => vsi0-sub2[10.240.64.5] : All Connections
+vsi1-sub1[10.240.0.4] => vsi0-sub3[10.240.128.4] : All Connections
+vsi1-sub1[10.240.0.4] => vsi1-sub2[10.240.64.4] : All Connections
+vsi1-sub1[10.240.0.4] => vsi1-sub3[10.240.128.5] : All Connections
+vsi1-sub2[10.240.64.4] => alb[Fake LB private IP][10.240.1.128] : All Connections
+vsi1-sub2[10.240.64.4] => alb[Fake LB private IP][10.240.1.4] : All Connections
+vsi1-sub2[10.240.64.4] => alb[Fake LB private IP][10.240.129.128] : All Connections
+vsi1-sub2[10.240.64.4] => alb[Fake LB private IP][10.240.65.128] : All Connections
+vsi1-sub2[10.240.64.4] => alb[LB private IP][10.240.65.4] : All Connections
+vsi1-sub2[10.240.64.4] => vsi0-ctrl-sub2[10.240.66.4] : All Connections
+vsi1-sub2[10.240.64.4] => vsi0-sub2[10.240.64.5] : All Connections
+vsi1-sub2[10.240.64.4] => vsi0-sub3[10.240.128.4] : All Connections
+vsi1-sub2[10.240.64.4] => vsi1-sub1[10.240.0.4] : All Connections
+vsi1-sub2[10.240.64.4] => vsi1-sub3[10.240.128.5] : All Connections
+vsi1-sub3[10.240.128.5] => alb[Fake LB private IP][10.240.1.128] : All Connections
+vsi1-sub3[10.240.128.5] => alb[Fake LB private IP][10.240.1.4] : All Connections
+vsi1-sub3[10.240.128.5] => alb[Fake LB private IP][10.240.129.128] : All Connections
+vsi1-sub3[10.240.128.5] => alb[Fake LB private IP][10.240.65.128] : All Connections
+vsi1-sub3[10.240.128.5] => alb[LB private IP][10.240.65.4] : All Connections
+vsi1-sub3[10.240.128.5] => vsi0-ctrl-sub2[10.240.66.4] : All Connections
+vsi1-sub3[10.240.128.5] => vsi0-sub2[10.240.64.5] : All Connections
+vsi1-sub3[10.240.128.5] => vsi0-sub3[10.240.128.4] : All Connections
+vsi1-sub3[10.240.128.5] => vsi1-sub1[10.240.0.4] : All Connections
+vsi1-sub3[10.240.128.5] => vsi1-sub2[10.240.64.4] : All Connections
+
```

Comment:
Add an explanation what is a fake private IP (vs. private ip)?

Reply:
where to add it?

Reply:
in the bottom? Near e.g. the explanation what a * stands for?

Reply:
its not the scope of this PR

Reply:
Shall we open an issue for this? Perhaps add it to an existing one? @adisos

Reply:
there is a plant to refactor all the messaging (statfeul, abstraction...) it should be part of it